### PR TITLE
doc: Added entry for run command option profile-create-new-if-missing

### PR DESCRIPTION
**Pull request by akhilpanchal**
@rpl 
Doc entry for [web-ext run command option `--profile-create-new-if-missing`](https://github.com/mozilla/web-ext/issues/2058). Follow up to [web-ext pull request](https://github.com/mozilla/web-ext/pull/2064).

Please let me know if I have missed something.

### CODE REVIEW

**Comment by akhilpanchal** [comment]
Hi @rpl:
Looks like this feature is [released in 5.4.0](https://github.com/mozilla/web-ext/issues/2058#issuecomment-732472310) but is [not documented](https://extensionworkshop.com/documentation/develop/web-ext-command-reference/)
Can you review these doc changes?

File: src/content/documentation/develop/web-ext-command-reference.md [hunk 1]
```diff
@@ -244,6 +245,12 @@ If a profile is not specified, it runs the extension using a new temporary profi
 
 Environment variable: `$WEB_EXT_FIREFOX_PROFILE`
 
+#### `--profile-create-if-missing`
+
+This option is to be used with `--firefox-profile` or `--chromium-profile`. With this option, the profile option provided will be treated as a directory path and if the directory does not exist, it will be created.
+
+Environment variable: `$WEB_EXT_PROFILE_CREATE_IF_MISSING`
+
```
Comment (rpl) [inline src/content/documentation/develop/web-ext-command-reference.md:253]: Nit, how about tweaking this description a bit to make it consistent with the --keep-profile-changes option that follows? e.g. something like:

```
With this option, the profile directory (specified by the `--firefox-profile` or `--chromium-profile` options) will be created if it does not exist yet. 

::: note alert
When this option is specified the `--firefox-profile` option is always treated as a directory path.
:::

Environment variable: `$WEB_EXT_PROFILE_CREATE_IF_MISSING`
```

**Review by rpl — COMMENTED** [review]
@akhilpanchal apologies for letting you wait that much on this PR, I have been away for a week and just got back.

The description of the new option is technically right, I just suggested some small tweaks to make it a bit more consistent with the description of the `--keep-profile-changes` option.
Let me know what do you think about the tweaks I did suggest in the inline comment below.

@caitmuenster would you mind to also review this from an editorial perspective after I've marked it as approved?

File: src/content/documentation/develop/web-ext-command-reference.md (hunk 1 above)
Comment (akhilpanchal) [reply src/content/documentation/develop/web-ext-command-reference.md:253]: Definitely! This is more concise and clear.

**Comment by caitmuenster** [comment]
Thanks so much for the patch, @akhilpanchal! This looks good from an editorial perspective.

File: src/content/documentation/develop/web-ext-command-reference.md [hunk 2]
```diff
@@ -23,9 +23,10 @@ contributors:
     sharang,
     chrisdavidmills,
     noraj,
+    akhilpanchal
   ]
-last_updated_by: noraj
-date: 2020-10-11 18:11:47
+last_updated_by: akhilpanchal
+date: 2020-11-15 12:46:15
```
Comment (rpl) [inline src/content/documentation/develop/web-ext-command-reference.md:29]: Nit, we may update the date before merging (not strictly mandatory to be fair)

**Review by rpl — APPROVED** [review]
@akhilpanchal Thanks a lot for following up the web-ext pull request by also updating the docs! :heart:

File: src/content/documentation/develop/web-ext-command-reference.md (hunk 2 above)
Comment (caitmuenster) [reply src/content/documentation/develop/web-ext-command-reference.md:29]: Hmm -- I'll go ahead and update it to reflect the next deploy date of Dec 03.

**Comment by caitmuenster** [comment]
Hey @akhilpanchal, I wanted to follow up with you but unfortunately couldn't find your contact information. :( I would love to send you some stickers as a small token of appreciation for your recent contributions! Would you mind sending me an email at cneiman [at] mozilla [dot] com with your current mailing address?

**Pull request merged**